Cargo.toml

```diff
@@ -1,14 +1,13 @@
 [package]
 name = "bevy_slippy_tiles"
-version = "0.4.3"
+version = "0.5.0"
 edition = "2021"
 description = "Provides slippy tile fetching functionality in the Bevy game engine"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/edouardpoitras/bevy_slippy_tiles"
 
 [dependencies]
-bevy = "0.12"
-futures-lite = "1.13"
```

Comment:
See the migration guide:

https://bevyengine.org/learn/migration-guides/0-12-to-0-13/#re-export-futures-lite-in-bevy-tasks